ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -248,6 +248,19 @@ class SimpleOperationTracker implements OperationTracker {
         // Here set the success target to the number of eligible replicas.
         replicaSuccessTarget = eligibleReplicas.size();
         break;
+      case ReplicateBlobOperation:
+        // Replicate one blob. Use the same "success target" and "request parallelism" config settings as PutOperation.
+        // But ReplicateBlobOperation sets crossColoEnabled to true.
+        // ON_DEMAND_REPLICATION_TODO: we may consider add its own configure setting if needed.
+        // ON_DEMAND_REPLICATION_TODO: may tune the order of the replica pool. Currently still local first and then remote.
+        // a. right now source HostNode is included. Probably will exclude it.
```

Comment:
The source host should be excluded, otherwise, we set up to at least have one success from the beginning.
If the source host is not excluded, then when the SimpleOperationTracker hits the source host, we know it's going to be a success. This would be equivalent to subtracting 1 from success target.

Reply:
I use the same parallelism and success target as Put. So parallelism is 3 and success target is 2. Basically it's retrying the Put Request and meet the Put quorum requirement. 
So even source host is included, we still will reach at least 2. 

But as you suggested, another way I can do is, exclude source host but set its configurations like success target as 1.

I'm ok with both but I prefer the current solution a little more. Because it's synced with Put requirement.

Reply:
ok, we can try this out and come back later.